docs/website/docs/user-guides/deploy/_editing_devfile.mdx

```diff
@@ -0,0 +1,177 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+Let's modify the `devfile.yaml` and add the respective deployment code.
+
+:::caution
+When copy/pasting to `devfile.yaml`, make sure the lines you inserted are correctly indented.
+:::
+
+`odo deploy` uses Devfile schema **2.2.0**. Change the schema to reflect the change:
+
+```yaml
+# Deploy "kind" ID's use schema 2.2.0+
+schemaVersion: 2.2.0
+```
+
+Add the `variables` section:
+
+<div>
+<CodeBlock language="yaml">
+{`
+# Add the following variables code anywhere in devfile.yaml
+# This MUST be a container registry you are able to access
+variables:
+  CONTAINER_IMAGE: quay.io/MYUSERNAME/`}{props.name}{`-odo-example
```

Comment:
Can you add a note about changing `MYUSERNAME` to an appropriate username?

Reply:
Done!